src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMappingTest.java

```diff
@@ -264,7 +264,8 @@ void getHearingChannels_ifAdjourmentEnabled_getNextHearing() {
                     .build())
                 .build())
             .dwpIsOfficerAttending(NO.getValue())
-            .adjournment(Adjournment.builder().typeOfNextHearing(AdjournCaseTypeOfHearing.TELEPHONE).build())
+            .adjournment(Adjournment.builder().typeOfNextHearing(AdjournCaseTypeOfHearing.TELEPHONE)
```

Comment:
Minor - add a newline here before .typeOfNextHearing so each aspect of the adjournment builder is on its own line